nupic/swarming/ModelRunner.py

```diff
@@ -242,24 +244,40 @@ def run(self):
     #Get field statistics from the input source
     fieldStats = self._getFieldStats()
     # -----------------------------------------------------------------------
-    # Construct the model instance
+    self._logger.error("Model %s RUNNING", self._modelID)
+    ## Construct the model instance
+
+    # special handling for global model sharing (for anomaly metrics,...)
+    # model.name needs to be overriden for swarming by modelID which is unique
+    # this needs to be called before model is created
+    swarmingName = id(self._modelID)
+    modelDescription['modelParams']['name']=swarmingName
+
     self._model = ModelFactory.create(modelDescription)
+    assert GlobalDict.get(swarmingName) is not None
+    print >>sys.stderr,"models=",str(GlobalDict())
```

Comment:
I will retest, but I think not. Actually printing to output is somehow
messy when combined with the swarming, standard print is ignored, I copied
this from other use in the code..